libs/surfaces/console1/console1.cc

```diff
@@ -175,6 +175,9 @@ Console1::begin_using_device ()
 	  f0 7d 20 00 00 00 01 00 7f 49 6f 6c 73 00 f7
 	*/
 
+	if (_in_use)
+		return 0;
+
```

Comment:
Why is this needed? `begin_using_device` cannot be called while the device is already active?
Best compare to `LaunchPadPro::begin_using_device`

Reply:
You are right. stop_using_device is called even if there is no connection, start_using_device is only called once. I will delete it.